.eslintrc.json

```diff
@@ -28,6 +28,7 @@
         "react/jsx-props-no-spreading":["warn"],
         "no-param-reassign":["off"],
         "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
-        "testing-library/prefer-screen-queries":["off"]
+        "testing-library/prefer-screen-queries":["off"],
+        "no-alert":"off"
```

Comment:
Why did you add this no-alert exception ?

Reply:
To prevent eslint warning to check other component's functionality. Forgot to remove it but now just removed and fixed it.